src/OrleansProviders/Streams/Memory/MemoryAdapterConfig.cs

```diff
@@ -10,17 +10,37 @@ namespace Orleans.Providers.Streams.Memory
     /// </summary>
     public class MemoryAdapterConfig
     {
+        /// <summary>
+        /// Stream provider name.
+        /// </summary>
         public String StreamProviderName { get; private set; }
 
+        /// <summary>
+        /// Total queue count name. Indicates the name of the property in provider config.
+        /// </summary>
         private const string TotalQueueCountName = "TotalQueueCount";
```

Comment:
the PopulateFromProviderConfig method is the only place that needs the name to get total queue count so far, do you think it might be needed somewhere else?